.github/workflows/ci.yml

```diff
@@ -66,17 +60,20 @@ jobs:
       run: python -m pytest -vv -rs -Wd
 
   test_wheels:
-    name: Wheel on linux
+    name: Wheel on Linux
     runs-on: ubuntu-latest
 
     steps:
     - uses: actions/checkout@v2
       with:
         submodules: recursive
 
-    - uses: pypa/cibuildwheel@v2.0.1
+    - uses: pypa/cibuildwheel@v2.1.1
       env:
-        CIBW_BUILD: cp39-manylinux_x86_64
+        CIBW_ARCHS: auto64
+        CIBW_BUILD: cp39-*
+        CIBW_BUILD_VERBOSITY: 2
+        CIBW_REPAIR_WHEEL_COMMAND: ""
```

Comment:
Why is this turned off? Wheels should always be "repaired", otherwise they are invalid anywhere other than the machine they were built on.

Reply:
It was the last error of the evening: the more extensive wheel-test in `wheels.yml` succeeded, but this single-platform check in `ci.yml` did not. I thought the `wheels.yml` might be running in a different container than the `ci.yml` version. I know from an explicit test that I can't `auditwheel repair` the wheels manually on my system (because of the platform variable).

My first thought was to drop the whole test (the version in `ci.yml`): it's tested in `wheels.yml`. However, `wheels.yml` is opt-in on ordinary builds.

Reply:
In the message for d4e37649ad4713c15cb5ef5fc0ca9f6c44f0a407, I should have said "reinstating" (by dropping the exclusion).

Reply:
Apparently, it worked. Great!